test/Notifier-test.ts

```diff
@@ -81,6 +81,7 @@ describe("Notifier", () => {
             decryptEventIfNeeded: jest.fn(),
             getRoom: jest.fn(),
             getPushActionsForEvent: jest.fn(),
+            supportsExperimentalThreads: jest.fn().mockReturnValue(false),
```

Comment:
Threads don't need to be on for these tests but `supportsExperimentalThreads()` needs to be defined